config/20.1/make.conf

```
@@ -63,7 +63,7 @@ net_freeradius3_UNSET=		HEIMDAL
 net_frr6_SET=			SNMP
 net_frr7_SET=			SNMP
 net_haproxy_SET=		LUA
-net_miniupnpd_SET=		CHECK_PORTINUSE PF_FILTER_RULES
+net_miniupnpd_SET=		CHECK_PORTINUSE  PF_FILTER_RULES
```

```suggestion
net_miniupnpd_SET=		CHECK_PORTINUSE PF_FILTER_RULES
```

whoopsie, shouldn't touch this line... stray whitespace